server/src/concepts/category/repositories/commands.js

```diff
@@ -0,0 +1,17 @@
+import Category from "../model/Category.js";
+
+const createCategory = async (data) => {
+  const newCategory = new Category({
+    ...data,
+  });
+
+  try {
+    await newCategory.save();
+    return newCategory;
+  } catch (err) {
+    console.log(err);
+    return "error";
```

Comment:
Why do you want to return string "error" instead of error? What is the meaning of this string?

Reply:
I just took it from an example, didn't pay attention, I'm already correcting it.